sdk/core/azure_core/Cargo.toml

```diff
@@ -28,8 +28,11 @@ serde_json.workspace = true
 sha2 = { workspace = true, optional = true }
 tokio = { workspace = true, optional = true }
 tracing.workspace = true
-typespec = { workspace = true, features = ["http", "json"] }
-typespec_client_core = { workspace = true, features = ["http", "json"] }
+typespec = { path = "../../typespec", features = ["http", "json"] }
```

Comment:
I think all crates should be using path dependencies until GA (or close). See previous comments.